src/app/entity/entity-mapper.service.ts

```diff
@@ -48,6 +48,29 @@ export class EntityMapperService {
     );
   }
 
+  /**
+   * Loads a list of Entity from the database whose IDs contain the prefix of the given resultEntity class.
+   *
+   * @param entityType the entity's class type (this would just be <code>Entity</code> for a pure entity).
+   * @returns A Promise containing an array with the entities.
+   */
+  public loadAll<T extends Entity>(entityType: { new(id: string): T; }): Promise<T[]> {
```

Comment:
I had actually tried to do a straight-forward type parameter like this for the `load()` function but couldn't get it to work.

`load` and `loadAll` should have similar behaviour, so I suggest we modify `load` to take the same style of parameter